torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1859,6 +1859,26 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            decorators=[_wrap_maybe_warns("floor_divide is deprecated, and will be removed")],
            supports_autograd=False,
            ),
+    UnaryUfuncInfo('frexp',
+                   op=torch.frexp,
+                   ref=np.frexp,
+                   dtypesIfCPU=floating_types_and(torch.half),
+                   dtypesIfCUDA=floating_types_and(torch.half),
+                   # skip testing torch.frexp as it is not supported by ROCm platform yet
+                   decorators=[skipCUDAIfRocm],
+                   test_inplace_grad=False,
```

Comment:
Set supports_tensor_out to False and could you revert the changes to test_out.py

Reply:
Thank you so much for the tip, the changes to `test_out.py` have been reverted in the latest PR.